modules/dataquery/jsx/welcome.tsx

```diff
@@ -892,7 +900,7 @@ function SingleQueryDisplay(props: {
       </div>
       {query.criteria ?
         <div>
-          <h3>Filters</h3>
+          <h3>{t('Filter', {ns: 'dataquery', count: 2})}</h3>
```

Comment:
You can use query.criteria to get the actual count of filters.

If it's indeterminate, my understanding is that 99 is better than 2 for languages that have multiple plural forms.